ProcessesApi.Tests/V1/E2ETests/Stories/UpdateSoleToJointProcessTests.cs

```diff
@@ -369,6 +369,58 @@ public void ProcessStateIsUpdatedToApplicationSubmitted()
                 .BDDfy();
         }
 
+        [Fact]
```

Comment:
Could we refactor test so that it is something like:
`[Theory]
[InlineData(SoleToJointStates.TenureInvestigationPassed)]
[InlineData(SoleToJointStates.TenureInvestigationFailed)]
[InlineData(SoleToJointStates.TenureInvestigationPassedWithInterview)]
        public void ProcessStateIsUpdatedToTenureInvestigationPassed(string newState)
        {
            this.Given(g => _processFixture.GivenASoleToJointProcessExists(SoleToJointStates.ApplicationSubmitted))
                    .And(a => _processFixture.GivenATenureInvestigationRequest(SoleToJointFormDataValues.Approve))
                .When(w => _steps.WhenAnUpdateProcessRequestIsMade(_processFixture.UpdateProcessRequest, _processFixture.UpdateProcessRequestObject, 0))
                .Then(a => _steps.ThenTheProcessStateIsUpdatedToTenureInvestigationPassed(_processFixture.UpdateProcessRequest))
                    .And(a => _steps.ThenTheProcessUpdatedEventIsRaised(_snsFixture, _processFixture.ProcessId, SoleToJointStates.ApplicationSubmitted, newState))
                .BDDfy();
        }`